Bonsai.Shaders/BindTexture.cs

```diff
@@ -58,35 +58,25 @@ public bool IndexSpecified
             get { return Index.HasValue; }
         }
 
-        IObservable<TSource> Process<TSource>(IObservable<TSource> source, Action<TSource> update)
+        /// <param name="source">The source sequence to be processed</param>
```

Comment:
This is really only here to make [CS1573](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs1573) happy. (Unfortunately it cares even if the method is private like this one.)

I'm sort-of not a fan of the XML documentation warnings because they encourage you to create useless self-evident comments like this.

Reply:
That's odd, on my vanilla VS community this doesn't get flagged as a warning somehow. If there are some editor settings we can toggle to turn this off it would be best, we really don't want to enforce having XML documentation comments for private method parameters.

Reply:
I tested on Visual Studio 2022 in a fresh root suffix (so completely default settings) and pinned the .NET SDK to either 6.0 or 8.0 stable using global.json and it still happens.

I want to emphasize it's this specific line and that it's CS1573 rather than [CS1591](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1591).

I *did* want to document `getTextureId` despite it being private because bare `Func`/`Action` delegates tend to get kinda hairy when they have many parameters since there's no parameter names. Plus private XML documentation can still be useful since it shows up in Intellisense:

![image](https://github.com/bonsai-rx/bonsai/assets/278957/b109edd6-497a-4332-a66f-ff9520b62fe6)